pkgs/applications/networking/cluster/terranix/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, ... }:
+
+stdenv.mkDerivation {
+
+  pname = "terranix";
+  version = "2.3.0";
+
+  src = fetchFromGitHub {
+    owner = "mrVanDalo";
+    repo = "terranix";
+    rev = "2.3.0";
+    sha256 = "030067h3gjc02llaa7rx5iml0ikvw6szadm0nrss2sqzshsfimm4";
+  };
+
+  installPhase = ''
+    mkdir -p $out/{bin,core,modules,lib}
+    mv bin core modules lib $out/
```

I think the arguments to `mkdir` do not make sense, as your `mv` immediately afterwards overwrite them. How about this simpler version?

```suggestion
    mkdir -p $out
    mv bin core modules lib $out/
```